File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        a = {}
        ans = []
        num1 = sorted(nums)
        for i,v in enumerate(num1):
            if v not in a:
                a[v] = i
        for i in nums:
            ans.append(a[i])
        return ans
            

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int n = nums.length;
        int num[] = new int[n];
        for(int i = 0;i<n;i++)
        {
            int count =0;
            for(int j=0;j<n;j++)
            {
                if(nums[i]>nums[j]  && j !=i)
                {
                    count++;
                }
            }
             num[i]=count;
        }
        return num;
    }
}